gapic/src/main/com/google/gapid/views/ShaderView.java

```diff
@@ -305,6 +307,9 @@ public void setShader(Service.Resource resource, API.Shader shader) {
           sourceTab.setText(shaderMessage.getSourceLanguage());
         }
       } else if (sourceTab != null) {
+        if (Arrays.asList(tabFolder.getSelection()).contains(sourceTab)) {
+          tabFolder.setSelection(spirvTab);
+        }
```

Comment:
Since we only have two tabs, I would just do an unconditional `tabFolder.setSelection(spirvTab)` here, but this is fine, too.

Reply:
I like simpler, I'll go ahead and push the change.